tests/psalm-baseline.xml

```diff
@@ -154,7 +149,8 @@
     </UndefinedClass>
   </file>
   <file src="lib/Service/RequestSignatureService.php">
-    <UndefinedClass occurrences="2">
+    <UndefinedClass occurrences="3">
+      <code>UUIDUtil</code>
       <code>UUIDUtil</code>
       <code>UUIDUtil</code>
     </UndefinedClass>
```

Comment:
Why do we use this section?

Reply:
UUIDUtil is a method from a class that don't exists on our side and because this the psalm return an error. Is a class from 3rdparty of server side and if we add this dependency to LibreSign composer.json could generate a conflict when the version of 3rdparty will be updated.

The psalm-baseline file is a file that map all the cases to be ignored.

Every when have an error reported by psalm that we want to ignore, we need to update the psalm baseline. Exists a script at composer.json with the command to update this.

Other moment that we need to do this is when we fix a problem that was ignored at psalm baseline.

Reply:
Thank u for the explanation. Resolving it.